### PR TITLE
update html & layout

### PR DESCRIPTION
**Pull request by tuturu9**

### CODE REVIEW

File: coronataiou/templates/base.html [hunk 1]
```diff
@@ -14,53 +14,54 @@
     {% else %}
     <title>Galileo Temperature Manager</title>
     {% endif %}
-    <style>
-    body {
-    padding-top: 60px;
-    }
```
Comment (joooooooooe-star) [inline coronataiou/templates/base.html:20]: does this render properly without the padding?

File: coronataiou/templates/base.html [hunk 2]
```diff
@@ -14,53 +14,54 @@
...
+        <img class="img-fluid rounded mb-3 mb-md-0" src="https://i.imgflip.com/52b2ut.gif" alt="">
+        <h3><a class="navbar-brand" href="/home">GALILEO CO. LTD.</a></h3>
         <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
               <span class="navbar-toggler-icon"></span>
             </button>
         <div class="collapse navbar-collapse" id="navbarResponsive">
           <ul class="navbar-nav ml-auto">
             <li class="nav-item active">
-              <a class="nav-link" href="/home">Home
+              <a class="nav-link" href="/home"><b>Home</b>
```
Comment (joooooooooe-star) [inline coronataiou/templates/base.html]: should use the {{url_for}} format

File: coronataiou/templates/base.html [hunk 3]
```diff
@@ -14,53 +14,54 @@
...
+            <li class="nav-item">
+              <a class="nav-link" href={{url_for("add_record")}}><b>Add</b></a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="/about">About</a>
+              <a class="nav-link" href="{{ url_for('edit_table') }}"><b>Update</b></a>
+            </li>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="/about"><b>About</b></a>
```
Comment (joooooooooe-star) [inline coronataiou/templates/base.html]: should use the {{url_for}} format

File: coronataiou/templates/base.html [hunk 4]
```diff
@@ -14,53 +14,54 @@
...
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="#">Contact</a>
+              <a class="nav-link" href="/contact"><b>Contact</b></a>
```
Comment (joooooooooe-star) [inline coronataiou/templates/base.html]: should use the {{url_for}} format

**Review by joooooooooe-star — CHANGES_REQUESTED** [review]

File: coronataiou/templates/base.html (hunk 1 above)
Comment (tuturu9) [reply coronataiou/templates/base.html:20]: yes it does render properly

**Review by joooooooooe-star — APPROVED** [review]
approved

**Pull request merged**